.github/workflows/ci.yml

```
@@ -167,6 +167,17 @@ jobs:
     permissions:
       checks: write
       pull-requests: write
+    strategy:
+      matrix:
+        tari_target_network: [
```

Maybe a better name choice
```suggestion
        tari_network_map: [
```